### ISSUE TITLE
LED Color for Groups

### ISSUE THREAD

**Issue by dgwood12** [chuot/rdio-scanner#179]
I just started using rdio-scanner today and am loving it so far! Thank you for your hard work!

In setting up, I thought that while it is great to have functionality to set LED color by talkgroup, I think it would also be helpful to be able to set by group. This way, you can set the color of a swath of talkgroups that are assigned to a group instead of color coding them individually. Functionality could be to make the talkgroups LED color setting priority. This way, say for example you have a law enforcement group set to blue but have 1 talkgroup in that group set to green. That talkgroup would still be set to green and all other talkgroups in that group that do not have an LED color setting assigned would get the group color (blue). Hope this makes sense.

Thanks!

**Comment by chuot** [comment]
Good idea, but it will go in the future v7 since it requires a database structural change.
